test/Node.test.js

```diff
@@ -27,6 +27,33 @@ describe('Node', function () {
             assert.strictEqual(Node._findSchema(schema, {}, path), schema.properties.child);
         });
 
+        it('should find schema inside an array item', function () {
+            var schema = {
+                properties: {
+                    job: {
+                        type: 'array',
+                        items: {
+                            properties: {
```

Comment:
You might want a `type: 'object'` above this line to make this valid JSON schema. Just for clarity.

Reply:
:+1: